src/components/column-sorting/column-sorting.component.ts

```diff
@@ -13,6 +11,8 @@ export class ColumnSortingComponent {
     @Input() state: ColumnSortingState;
     @Input() sortKey: string;
     @Input() orderNumber: number;
```

Comment:
Could we run a code tidy on this file as some of the indentation is incorrect. (Alt+Shift+F in VS Code should do it for you)